deploy/template.yaml

```diff
@@ -717,6 +718,8 @@ Resources:
             TableName: !Ref SessionsTable
         - DynamoDBWritePolicy:
             TableName: !Ref ClientOAuthSessionsTable
+        - DynamoDBCrudPolicy:
+            TableName: !Ref UserIssuedCredentialsV2Table
```

Comment:
I think this overlaps with https://github.com/govuk-one-login/ipv-core-back/pull/1570

cc @MikeCollingwood